### ISSUE TITLE
Feature: Predefined "Menu Item Visibility" Options (for kids, for example)

### ISSUE THREAD

**Issue by nicolasmafra** [libretro/RetroArch#13167]
It would be useful if retroarch had an alternative, basic menu, so that users new to emulation would find it easier to use retroarch.

In this case it would have an option to switch between the basic menu and the advanced menu.

Example: a novice user would just like to select the game and play. Of course, if he just downloaded retroarch, he would need to download cores and import the ROMs. Any other option I consider advanced.

**Comment by nickmafra** [comment]
Yes, but Kiosk mode don't allow install cores and import content.

**Comment by nickmafra** [comment]
Note: the Goal of "amateur mode" is to someone install RetroArch, download core, import content and play games, without another "more complicated" configuration items on the screeen. That's why it could be the default menu item visibility too.
